app/gateway/2.7.x/admin-api/consumer-groups/examples.md

```diff
@@ -0,0 +1,532 @@
+---
+title: Consumer Groups Examples
+badge: enterprise
+---
+
+With consumer groups, you can define any number of rate limiting tiers and
+apply them to subsets of consumers, instead of managing each consumer
+individually.
+
+For example, you could define three consumer groups:
+* A "gold tier" with 1000 requests per minute
+* A "silver tier" with 10 requests per second
+* A "bronze tier" with 6 requests per second
+
+The `consumer_groups` endpoint works together with the [Rate Limiting Advanced plugin](/hub/kong-inc/rate-limiting-advanced).
+
+Consumers that are not in a consumer group default to the Rate Limiting advanced
+plugin’s configuration, so you can define tier groups for some users and
+have a default behavior for ungrouped consumers.
+
+To use consumer groups for rate limiting, you need to:
+* Create one or more consumer groups
+* Create consumers
+* Assign consumers to groups
+* Configure the Rate Limiting Advanced plugin with the `enforce_consumer_groups`
+and `consumer_groups` parameters, setting up the list of consumer groups that
+the plugin will accept
+* Configure rate limiting for each consumer group, overriding the plugin's
+configuration
+
+For all possible requests, see the
+[Consumer Groups reference](/gateway/{{page.kong_version}}/admin-api/consumer-groups/reference).
+
+## Set up consumer group
+
+1. Create a consumer group named `JL`:
+
+{% capture create_consumer_group %}
+{% navtabs codeblock %}
+{% navtab cURL %}
+```bash
+curl -i -X POST http://{HOSTNAME}:8001/consumer_groups \
+  --data name=JL
+```
+{% endnavtab %}
+{% navtab HTTPie %}
+```bash
+http POST :8001/consumer_groups name=JL
+```
+{% endnavtab %}
+{% endnavtabs %}
+{% endcapture %}
+
+{{ create_consumer_group | indent | replace: " </code>", "</code>" }}
+
+    Response:
+
+    ```json
+    {
+        "created_at": 1638915521,
+        "id": "8a4bba3c-7f82-45f0-8121-ed4d2847c4a4",
+        "name": "JL"
+    }
+    ```
+
+2. Create a consumer, `DianaPrince`:
+
+{% capture create_consumer %}
+{% navtabs codeblock %}
+{% navtab cURL %}
+```bash
+curl -i -X POST http://{HOSTNAME}:8001/consumers \
+  --data username=DianaPrince
+```
+{% endnavtab %}
+{% navtab HTTPie %}
+```bash
+http POST :8001/consumers username=DianaPrince
+```
+{% endnavtab %}
+{% endnavtabs %}
+{% endcapture %}
+
+{{ create_consumer | indent | replace: " </code>", "</code>" }}
+
+    Response:
+
+    ```json
+    {
+        "created_at": 1638915577,
+        "custom_id": null,
+        "id": "8089a0e6-1d31-4e00-bf51-5b902899b4cb",
+        "tags": null,
+        "type": 0,
+        "username": "DianaPrince",
+        "username_lower": "dianaprince"
+    }
+    ```
+
+3. Add `DianaPrince` to the `JL` consumer group:
```

Comment:
Do we want to show an example of adding multiple consumers to a group?
If so, we need to also notify user that we will validate the full list first before assigning:
Consumers need to be valid (existing, and not already in the group)
The assign will error out and error message will contain first element that doesn't comply, no consumer would be assigned in this case.

Reply:
Added a separate example, see https://github.com/Kong/docs.konghq.com/pull/3443/commits/57813ee86dbda51b3ac8485f5bcab327e92c57e4